*** Starting uWSGI 2.0.17.1 (64bit) on [Sat Jul 21 15:37:00 2018] ***
compiled with version: 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.2) on 21 July 2018 12:45:58
os: Darwin-17.6.0 Darwin Kernel Version 17.6.0: Tue May  8 15:22:16 PDT 2018; root:xnu-4570.61.1~1/RELEASE_X86_64
nodename: Tungs-MBP
machine: x86_64
clock source: unix
detected number of CPU cores: 4
current working directory: /Volumes/Data/Projects/PythonExamples/Flask/Flask_Chapter6
detected binary path: /Volumes/Data/Projects/PythonExamples/Flask/Flask_Chapter6/venv/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 709
your memory page size is 4096 bytes
detected max file descriptor number: 4864
lock engine: OSX spinlocks
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 0.0.0.0:8000 fd 3
Python version: 3.7.0 (default, Jun 29 2018, 20:13:13)  [Clang 9.1.0 (clang-902.0.39.2)]
Set PythonHome to venv/
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x7f9d3b501640
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 72888 bytes (71 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x7f9d3b501640 pid: 14684 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 14684, cores: 1)
*** Starting uWSGI 2.0.17.1 (64bit) on [Sat Jul 21 15:38:37 2018] ***
compiled with version: 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.2) on 21 July 2018 12:45:58
os: Darwin-17.6.0 Darwin Kernel Version 17.6.0: Tue May  8 15:22:16 PDT 2018; root:xnu-4570.61.1~1/RELEASE_X86_64
nodename: Tungs-MBP
machine: x86_64
clock source: unix
detected number of CPU cores: 4
current working directory: /Volumes/Data/Projects/PythonExamples/Flask/Flask_Chapter6
detected binary path: /Volumes/Data/Projects/PythonExamples/Flask/Flask_Chapter6/venv/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 709
your memory page size is 4096 bytes
detected max file descriptor number: 4864
lock engine: OSX spinlocks
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 0.0.0.0:8000 fd 3
Python version: 3.7.0 (default, Jun 29 2018, 20:13:13)  [Clang 9.1.0 (clang-902.0.39.2)]
Set PythonHome to venv/
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x7f9065403a30
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 72888 bytes (71 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x7f9065403a30 pid: 14705 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 14705, cores: 1)
[2018-07-21 15:38:47,739] ERROR in app: Exception on / [GET]
Traceback (most recent call last):
  File "/Volumes/Data/Projects/PythonExamples/Flask/Flask_Chapter6/venv/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Volumes/Data/Projects/PythonExamples/Flask/Flask_Chapter6/venv/lib/python3.7/site-packages/flask/app.py", line 1808, in full_dispatch_request
    self.try_trigger_before_first_request_functions()
  File "/Volumes/Data/Projects/PythonExamples/Flask/Flask_Chapter6/venv/lib/python3.7/site-packages/flask/app.py", line 1855, in try_trigger_before_first_request_functions
    func()
  File "./app.py", line 21, in create_tables
    db.create_all(app=app)
NameError: name 'db' is not defined
[pid: 14705|app: 0|req: 1/1] 127.0.0.1 () {34 vars in 725 bytes} [Sat Jul 21 15:38:47 2018] GET / => generated 291 bytes in 10 msecs (HTTP/1.1 500) 2 headers in 84 bytes (2 switches on core 0)
[2018-07-21 15:38:48,061] ERROR in app: Exception on /favicon.ico [GET]
Traceback (most recent call last):
  File "/Volumes/Data/Projects/PythonExamples/Flask/Flask_Chapter6/venv/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Volumes/Data/Projects/PythonExamples/Flask/Flask_Chapter6/venv/lib/python3.7/site-packages/flask/app.py", line 1808, in full_dispatch_request
    self.try_trigger_before_first_request_functions()
  File "/Volumes/Data/Projects/PythonExamples/Flask/Flask_Chapter6/venv/lib/python3.7/site-packages/flask/app.py", line 1855, in try_trigger_before_first_request_functions
    func()
  File "./app.py", line 21, in create_tables
    db.create_all(app=app)
NameError: name 'db' is not defined
[pid: 14705|app: 0|req: 2/2] 127.0.0.1 () {34 vars in 704 bytes} [Sat Jul 21 15:38:47 2018] GET /favicon.ico => generated 291 bytes in 319 msecs (HTTP/1.1 500) 2 headers in 84 bytes (2 switches on core 0)
*** Starting uWSGI 2.0.17.1 (64bit) on [Sat Jul 21 15:43:11 2018] ***
compiled with version: 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.2) on 21 July 2018 12:45:58
os: Darwin-17.6.0 Darwin Kernel Version 17.6.0: Tue May  8 15:22:16 PDT 2018; root:xnu-4570.61.1~1/RELEASE_X86_64
nodename: Tungs-MBP
machine: x86_64
clock source: unix
detected number of CPU cores: 4
current working directory: /Volumes/Data/Projects/PythonExamples/Flask/Flask_Chapter6
detected binary path: /Volumes/Data/Projects/PythonExamples/Flask/Flask_Chapter6/venv/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 709
your memory page size is 4096 bytes
detected max file descriptor number: 4864
lock engine: OSX spinlocks
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 0.0.0.0:8000 fd 3
Python version: 3.7.0 (default, Jun 29 2018, 20:13:13)  [Clang 9.1.0 (clang-902.0.39.2)]
Set PythonHome to venv/
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x7fafc95017e0
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 72888 bytes (71 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x7fafc95017e0 pid: 14755 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 14755, cores: 1)
